README.md

```diff
@@ -191,6 +191,12 @@ This component receives the following props :
 - `props.selectionIndex`
   - The index of the highlighted option for rendering
 
+#### props.skipBlankSearch
```

Comment:
I think it's easier to reason about complex options when their defaults are false.

What do you think about `showOptionsWhenEmpty` ?